### ISSUE TITLE
 Filters in request log

### ISSUE THREAD

**Issue by uzhosnah** [RequestPolicyContinued/requestpolicy#655]
It will be good to have 4 checkboxes, that re-filter request log
- blocked by user rule,
- blocked by default,
- allowed by user rule,
- allowed by default.

Also "blocked by user" and "blocked by default" can be highlighted to different colors.
All this will help to filter out whitelisted and blacklisted entries and focus only on requests, not described in rules.

**Comment by nodiscc** [comment]
Duplicate of https://github.com/RequestPolicyContinued/requestpolicy/issues/157
